app/containers/AudioPlayer/index.test.tsx

```
@@ -0,0 +1,656 @@
+
+			render(<AudioPlayer {...defaultProps} />);
+
+			await act(async () => {});
+
+			expect(AudioManager.setOnPlaybackStatusUpdate).not.toHaveBeenCalled();
+			expect(AudioManager.setRateAsync).not.toHaveBeenCalledWith('mock-audio-key', expect.any(Number));
+		});
+	});
+
+	describe('InteractionManager task cleanup', () => {
+		it('cancels the InteractionManager task when the component unmounts', async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)